plugins/opensource/directionality.md

```diff
@@ -12,10 +12,6 @@ controls: toolbar button
 
 This plugin adds directionality controls to the toolbar, enabling {{site.productname}} to better handle languages written from right to left. It also adds a toolbar button for each of its values, `ltr` for left-to-right text and `rtl` for right-to-left text.
 
-**Type:** `String`
-
-**Possible Values:** `ltr`, `rtl`
-
```

Comment:
For this case would it be clearer to state

**Property** `toolbar`
**Type:** `String`	
**Possible Values:** `ltr`, `rtl`

having the property omitted is the confusing part as u need to imply it to toolbar

Reply:
That part is covered in the next section I believe: https://www.tiny.cloud/docs/plugins/opensource/directionality/#toolbarbuttons